js/components/HomeNavigationTab/styles.js

```diff
@@ -0,0 +1,23 @@
+import { StyleSheet, Dimensions } from 'react-native';
+
+var { height, width } = Dimensions.get('window');
+
+export const styles = StyleSheet.create({
+  tab: {
+    padding: 4,
+    marginBottom: 10,
+    borderWidth: 2,
+    width: width / 2.2,
+    height: height / 5.16363636364,
```

Comment:
The number is the multiplier for varying screen widths, so depending on screens with bigger sizes, it will divide by that number to give the same proportions as the iPhone SE screen